docs/terms.md

```diff
@@ -0,0 +1,26 @@
+# What are the terms we use and what do they mean?
+
+### Group
+A group is a collection of unit files tied together. This can be seen as
+equivalent to kubernetes pods. Units within a Inago group are scheduled
```

Comment:
The thing is, Inago does not enforce the "scheduled on the same host". It's one of the assumptions we do. Not sure how to incorporate this here though.